starlette_web/contrib/constance/__init__.py

```diff
@@ -0,0 +1,86 @@
+from typing import Any, List, Dict
+
+from starlette_web.common.http.exceptions import (
+    NotSupportedError,
+    BaseApplicationError,
+    UnexpectedError,
+)
+from starlette_web.common.utils import import_string
+from starlette_web.contrib.constance.backends.base import BaseConstanceBackend
+
+from starlette_web.common.conf import settings
+
+
+class LazyConstance:
+    _backend: BaseConstanceBackend
+    _is_setup: bool
+
+    def __init__(self):
+        self._is_setup = False
+
+    async def get(self, key: str) -> Any:
+        self._setup()
+
+        if key not in settings.CONSTANCE_CONFIG:
+            raise NotSupportedError
+
+        value = await self._backend.get(key)
+        return self._postprocess_value(key, value)
+
+    async def set(self, key: str, value: Any) -> None:
+        self._setup()
+
+        if key not in settings.CONSTANCE_CONFIG:
+            raise NotSupportedError
+
+        expected_type = settings.CONSTANCE_CONFIG[key][2]
+        if type(value) != expected_type:
+            try:
+                # This is for cases like int-float, str-int and such
+                value = expected_type(value)
+            except (TypeError, ValueError) as exc:
+                raise NotSupportedError(details=str(exc))
+            except Exception as exc:
+                # I.e. if we try to pass datetime to uuid value, which causes OverflowError
+                raise UnexpectedError(details=str(exc))
+
+        await self._backend.set(key, value)
+
+    async def mget(self, keys: List[str]) -> Dict[str, Any]:
+        self._setup()
+
+        if not keys:
+            return {}
+
+        for key in keys:
+            if key not in settings.CONSTANCE_CONFIG:
+                raise NotSupportedError
```

Comment:
```suggestion
                raise NotSupportedError(
                    f"Key {key} is not supported for configured constance"
                )
```

ну и везде ниже, можешь тоже плиз запроввадить такое сообщение или подобное (думаю было бы круто в ошибке видеть какой именно key не найден)

Reply:
добавил, только сообщение чуть другое